sdk/cosmos/azure-spring-data-cosmos-test/src/test/java/com/azure/spring/data/cosmos/repository/integration/CompositeIndexIT.java

```diff
@@ -51,6 +53,7 @@ public void setup() {
 
     @Test
     public void canSetCompositeIndex() {
+        collectionManager.deleteContainer(information);
```

Comment:
why do we ned to delete container in these two tests?

Reply:
We do because the tests don't always run in the same order, and depending on the order the container will already be created for one of them and it throws off the logic testing.